# Reference: http://www.sogo-unyu.co.jp/useful/track/

wheel_radius: 0.555
wheel_width: 0.600
trailer_width: 5.115
trailer_depth: 2.480
trailer_height: 0.655
hitch_length: 0.065
tractor_width: 5.500
tractor_depth: 2.480
tractor_height: 2.235
